### PR TITLE
fix(atomic): fix layout issue on product variants with imageSize set to none

### PR DESCRIPTION
**Pull request by fpbrault**
This PR fixes the product variant icons from taking the full width of the product element when the imageSize is set to none

image

https://coveord.atlassian.net/browse/KIT-3608

### CODE REVIEW

**Review by alexprudhomme — COMMENTED** [review]
The before after pictures seem to be wrong. when imageSize is none there should not be image.

**Comment by fpbrault** [comment]
> The before after pictures seem to be wrong. when imageSize is none there should not be image.

The before/after are not for imageSize none, but instead for imageSize small, to show the layout change

File: packages/atomic/src/components/commerce/atomic-product/atomic-product.pcss [hunk 1]
```diff
@@ -18,7 +18,7 @@
         &.image-large {
           atomic-product-section-children .product-child {
             @mixin aspect-ratio-h 1 / 1, auto;
-            width: 33%;
+            width: 25%;
```
Comment (alexprudhomme) [inline packages/atomic/src/components/commerce/atomic-product/atomic-product.pcss:21]: Why this change as well ?

File: packages/atomic/src/components/common/template-system/cell-desktop.pcss [hunk 2]
```diff
@@ -4,11 +4,11 @@
   grid-template-areas:
     'badges'
     'visual'
-    'children'
     'title'
     'title-metadata'
     'emphasized'
     'excerpt'
+    'children'
```
Comment (alexprudhomme) [inline packages/atomic/src/components/common/template-system/cell-desktop.pcss:11]: What does this do ?

Comment (fpbrault) [reply packages/atomic/src/components/common/template-system/cell-desktop.pcss:11]: puts the variants at the bottom, preventing the layout shifts when some products dont have variants. This is what is shown in the before/after images

Comment (fpbrault) [reply packages/atomic/src/components/common/template-system/cell-desktop.pcss:11]: @alexprudhomme actually gonna move these two changes in a separate PR I think

**Review by alexprudhomme — APPROVED** [review]
